README.md

```diff
@@ -38,28 +38,30 @@ To find out what's new in this release, please read:
 
 Installation
 ------------
-If you have superuser power, you can install MyHDL as follows:
+It is recommended to install MyHDL (and your project's other dependencies) in
+a virtualenv.
```

Comment:
Should include a link to virtualenv.

```[virtualenv](https://packaging.python.org/guides/installing-using-pip-and-virtualenv/)```
[virtualenv](https://packaging.python.org/guides/installing-using-pip-and-virtualenv/)

Reply:
how would you feel about linking to this instead? https://docs.python-guide.org/dev/virtualenvs/

It recommends `pipenv`, which in my opinion is currently the best tool for managing a project's dependencies, but also explains the other options such as directly creating a virtualenv or using virtualenvwrapper.